### DISCUSSION TITLE
Song Select: Display beatmaps from not working

### DISCUSSION THREAD

**Discussion by MrFastDie** [ppy/osu#14941] [Q&A]
In my settings I have decided that I only want to see maps with a higher star rating than 4.5

I use the F5 Button to play random maps and plenty of 3* maps are showing. 

I did not spent too much investigation in it but I have the HD mod enabled aswell and I also quit back to the main menu and gone back to singleplayer.

The expected behavior is not seeing maps with a higher rating than 4.5* at all.

**Comment by peppy** [comment]
Cannot reproduce. Provide logs, video, something.

**Reply by integerrr** [reply]
Managed to reproduce on my end.
[network.log](https://github.com/ppy/osu/files/7274335/network.log)
[runtime.log](https://github.com/ppy/osu/files/7274336/runtime.log)

It seems to resolve itself after I "refreshed" it by toggling `show converted` but it only works 1 beatmap set at a time i.e. I have to toggle `show converted` for every mapset that doesn't get filtered by SR.

https://user-images.githubusercontent.com/7279624/135767352-a65c0f45-fc14-400f-b173-5e1b6b6ef086.mp4

**Reply by peppy** [reply]
You're using "next track". That doesn't get filtered at all. Please try using *random* instead.

**Reply by MrFastDie** [reply]
> You're using "next track". That doesn't get filtered at all. Please try using _random_ instead.

regardless I think "next track" should get filtered aswell cause with that setting I don't want to see any stars below my set value.

**Reply by peppy** [reply]
using next track means you're changing track on the global music player. this doesn't have the concept of filtering. may i ask why you are using that?

**Comment by MrFastDie** [comment]
Version is: Lazer 2021.927.0

You have to watch the vid till the end - I had a wrong track at first but ull see the last one is a 2*

https://www.youtube.com/watch?v=jxytxveAWeY&ab_channel=MrFastDie


The quality is probably garbage as fuck - but I dont have smth other than my mobile phone to record off atm since my usual PC died and this one is very week. If you need another vid tell me I'll try to get my hands on one.

Not sure if thats from interest but I'm using void linux and used the osu!lazer installation provided by the lutris launcher. I could probably try to show you something on discord if ur interested but on my old "faster" pc that hangs aswell so it could be crap aswell...

**Reply by peppy** [reply]
You're using a "next track" binding. Please use the "random" binding instead.
